transport/transport-runtime/code_gen_cfg.textproto

```diff
@@ -0,0 +1,3 @@
+vendor_package: "com.google"
```

Comment:
you should use a more specific vendor package, i.e. `com.google.android.datatransport.runtime`, otherwise we will be prone to symbol conflicts in the future